### DISCUSSION TITLE
OCAD map with errors

### DISCUSSION THREAD

**Discussion by fimich1** [OpenOrienteering/mapper#2195] [Questions & Answers] [answered]
My ocad map contein some errors, i can open it in Openorienteerin mapper, but after changing and drawing i can/t save it for OCAD format
![изображение_2023-12-12_222057756](https://github.com/OpenOrienteering/mapper/assets/38435351/86db5898-72bd-409f-80bd-b02419daa1f3)
![image](https://github.com/OpenOrienteering/mapper/assets/38435351/937b5c03-a6c8-43e8-8d81-2ce47262bc63)
should i change enything?

**Comment by dl3sdo** [comment]
This is a well-known bug, see #2118 and #2185 
There is a description of a workaround.

**Comment by dl3sdo** [comment, marked as answer]
Closed by https://github.com/OpenOrienteering/mapper/pull/2035 in [v0.9.6](https://github.com/OpenOrienteering/mapper/releases/tag/v0.9.6)
